Mathlib/CategoryTheory/NatIso.lean

```diff
@@ -100,12 +100,16 @@ theorem trans_app {F G H : C ⥤ D} (α : F ≅ G) (β : G ≅ H) (X : C) :
     (α ≪≫ β).app X = α.app X ≪≫ β.app X :=
   rfl
 
-theorem app_hom {F G : C ⥤ D} (α : F ≅ G) (X : C) : (α.app X).hom = α.hom.app X :=
+theorem hom_app {F G : C ⥤ D} (α : F ≅ G) (X : C) : (α.app X).hom = α.hom.app X :=
```

Comment:
There is an issue with naming conventions here. `hom` is a field of the `Iso` structure, and the automatically generated lemma by the simps attribute for the `app` definition is `app_hom`, not `hom_app`. (Actually, `NatIso.app_hom` is a duplicate of `Iso.app_hom`, so that this lemma could be removed.)

Reply:
Thanks for catching this.